README.md

```diff
@@ -224,31 +224,15 @@ docker build -t retdec:dev . -f Dockerfile.dev
 ```
 
 ### Run Container
-
-To decompile a binary, create a container to upload the binary to:
-```
-docker create --name retdec_init retdec
-```
-
-Upload the binary (note the destination directory should be a directory with read/write permissions, such as `/home/retdec/`):
-```
-docker cp <file> retdec_init:/destination/path/of/binary
+Make sure the folder writable by retdec
 ```
-
-Commit the copied files into the container image:
+chmod -R 0666 /path/to/local/folder
```

Comment:
Are the permissions correct? `0666` makes the directory non-accessible. To enter a directory, it must have the `x` bit set, and `0666` removes that bit. So, after a decompilation, you will be unable to read the results.

Reply:
im not sure about this :| Its still accesible for me

Reply:
```
$ mkdir test
$ touch test/file.txt
$ chmod -R 0666 test
chmod: cannot access 'test/file.txt': Permission denied
$ cat test/file.txt
cat: test/file.txt: Permission denied
```
I suggest using `0777` instead of `0666`, and changing only permissions on the directory by removing the `-R` parameter.